_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,283 @@
+- [HTML in ARIA](https://www.w3.org/TR/html-aria/)
+- [Implicit WAI-ARIA Semantics](https://www.w3.org/TR/wai-aria-1.1/#implicit_semantics)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Element `ul` with role `list` has at least one child node `span` with the required owned element `listitem` as an explicit semantic role.
+
+```html
+<ul role='list'>
+	<span role='listitem'>
+	</span>
+	<span>
+	</span>
+</ul>
+```
+
+#### Passed example 2
+
+Element `div` with role `tablist` has child node `span` with the required owned element `tab` as an explicit semantic role.
+
+```html
+<div role='tablist'>
+	<span role='tab'>
+	</span>
+</div>
+```
+
+#### Passed example 3
+
+Multiple roles that each have an instance of their required owned elements present as child nodes.
+
+```html
+<table role='grid'>
+	<tr role='row'>
+		<span role='cell'>
+		</span>
+	</tr>
+</table>
+```
+
+#### Passed example 4
+
+Multiple types of required owned elements are present.
+
+```html
+<ul role='menu'>
+	<li></li> <!-- implicit role -->
+	<li role='menuitem'></li>
+	<li role='menuitemradio'></li>
+</ul>
+```
+
+#### Passed example 5
+
+Required owned element does not have to be an immediate child of the role, but could be any descendant. (See accessibility support notes for details).
+
+```html
+<div role="table">
+	<div>
+		<div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 6
+
+Nested required owned elements.
+
+```html
+<div role="grid">
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 7
+
+Element with role `list` contains a required owned element which has an implicit role of `listitem`.
+
+```html
+<div role='list'>
+   <li>
+   </li>
+</div>
+```
+
+#### Passed example 8
+
+Element with role `menubar` can have multiple required owned element (`menuitem` `menuitemcheckbox` and `menuitmeradio`) but only one is necessary.
+
+```html
+<div role='menubar'>
+   <div role='menuitem'>
+   </div>
+   <div role='menuitemcheckbox'>
+   </div>
+	 <div role='menuitmeradio'>
+   </div>
+</div>
+```
+
+#### Passed example 9
+
+Element with role `list` contains a required `listitem`, the relationship between both elements is made possible through the `aria-owns` properties.
+
+**Note:** This test case follows the definition of [owned by](#owned-by) used in this rule. If implemented differently, this definition could cause differences in outcome for this test case.
```

Comment:
This is the definition of "owned by" made by WAI ARIA, not just "used in this rule" (ie, it is not something we come up ourselves, it is the actual definition and if a browser/AT does not respect this definition, it's the browser/AT fault for not respecting WAI ARIA, not our fault for making weird definition).

I think this should rather link to the definition of "owned by" in WAI ARIA" (same as "Owned element" in the background Section).

Reply:
Because myself and @annethyme started authoring this rule (owned elements) and the [context rule](https://github.com/act-rules/act-rules.github.io/pull/255), after a long discussion of what definition to use for owned elements which happened here - https://github.com/act-rules/act-rules.github.io/issues/302, a decision was made to author our own. 

The definition is authored here - https://github.com/act-rules/act-rules.github.io/pull/255/files#diff-e8efc5e044aa522fbb04b4f072890fa3